clusterloader2/pkg/measurement/common/etcd_metrics.go

```diff
@@ -184,7 +184,8 @@ func (e *etcdMetricsMeasurement) stopAndSummarize(host, provider string, port in
 
 func (e *etcdMetricsMeasurement) getEtcdMetrics(host, provider string, port int) ([]*model.Sample, error) {
 	// Etcd is only exposed on localhost level. We are using ssh method
-	if provider == "gke" {
+	// TODO(ace): --managed flag instead of provider switch?
```

Comment:
I'd rather create a new field in ClusterLoaderConfig: `isSshSupported` that would be initialized depending on the provider. Then we could have a generic condition for disabling things that require ssh

Reply:
Actually ClusterConfig would be a better place

Reply:
moved to cluster config and used the value throughout. I left the prometheus one as individually checking the values since it's not technically the same condition -- let me know if you think some more cleanup is appropriate there.